cmd/inspect.go

```diff
@@ -223,9 +222,6 @@ func (cli *CLI) setupRunners(opts Options, dir string) (*tflint.Runner, []*tflin
 	}
 	annotations := map[string]tflint.Annotations{}
 	for path, file := range files {
-		if !strings.HasSuffix(path, ".tf") {
-			continue
-		}
```

Comment:
This also needs to be fixed.
https://github.com/terraform-linters/tflint/blob/fcbcce59c42d22c8ea69697c3c81ae85ee941413/langserver/handler.go#L181-L183